sql/item.cc

```diff
@@ -11211,7 +11310,8 @@ void Item_direct_view_ref::update_used_tables()
 
 table_map Item_direct_view_ref::used_tables() const
 {
-  DBUG_ASSERT(fixed());
+  if(!fixed())
+    return (table_map) 0;
```

Comment:
once again, we can end up calling this during setup_fields() so the item may no (yet) be fixed.